synapse/cortex.py

```diff
@@ -1584,6 +1584,22 @@ async def initServiceRuntime(self):
 
         await self._initStormSvcs()
 
+        # # Check for deprecated properties which are unused and unlocked
+        deprs = await self.getDeprLocks()
+        for propname, locked in deprs.items():
+            if locked:
+                continue
+
+            count = 0
+            for layr in self.layers.values():
+                count += await layr.getPropCount(propname)
+                if count:
+                    break
+
```

Comment:
This is probably going to be fast enough that it won't matter either way but you could use a for/else loop and avoid needing to use count here.

Reply:
I always forget about for..else because it's such a weird convention. 

Will that work here? Won't that only apply if there aren't any iterations through the layers?

Reply:
Yeah its not a convention that gets used super often but it does come in handy once in a while. The way you could use it here would be:
```
for layr in self.layers.values():
    if await layr.getPropCount(propname):
        break
else:
    mesg = f'Deprecated property {propname} is unlocked and not in use. Recommend locking.'
    logger.warning(mesg)
```

Reply:
That does look a bunch cleaner. Thanks for the tip!